### PR TITLE
Minor changes

### LINKED ISSUE #1204: (Not an issue) Create a link for patreon in Xenia´s webpage
Sorry for opening this, but it´s only intended to propose to @DrChat and @gibbed one thing: Add a link to the Xenia´s patreon in the official web of xenia.

I found (By chance) the patreon of Xenia, and I think more people can contribute in the patreon´s page if a link is added in the official page such as other emulators do (Cemu, rpcs3, etc) I 've been following this emulator since more than a year, and I discovered the patreon´s page today...

### PR DESCRIPTION
**Pull request by Margen67**
You can see a preview here: https://margen67.github.io/xenia/

Closes #1204.

### CODE REVIEW

**Comment by gibbed** [comment]
Why is this all in a single commit?

**Comment by Margen67** [comment]
@gibbed Should it not be?

**Comment by gibbed** [comment]
You listed a whole bunch of unrelated changes in the PR, it's fine for modifications to gh-pages to be rolled up into a single PR like this, but the actual changes should still be grouped into relevant commits.

**Comment by Margen67** [comment]
How should the commits be grouped/sorted?

**Comment by gibbed** [comment]
Relationship? Rationale? Use your best judgement?

**Comment by Margen67** [comment]
@gibbed I seperated the commits.

File: _config.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ description: >
   games on modern PCs.
 baseurl:
 url: "https://xenia.jp"
-twitter_username: xeniaproject
+patreon_username: xenia_project
```
Comment (gibbed) [inline _config.yml]: It's fine if you want to replace the Twitter link with a Patreon link, but I think the existing code to add a link to Twitter shouldn't be removed. Just comment out the variable that enables it instead.

File: _config.yml [hunk 2]
```diff
@@ -57,9 +57,6 @@ navigation:
     - title: YouTube
       url: https://www.youtube.com/channel/UC6bJhkUUHE18HVG2mpbygSQ
       icon: videocam
-#    - title: "#xenia @ freenode"
```
Comment (gibbed) [inline _config.yml]: No need to remove this.

File: _config.yml [hunk 3]
```diff
@@ -71,14 +68,11 @@ navigation:
     - title: Issues
       url: https://github.com/xenia-project/xenia/issues
       icon: bug_report
-#    - title: Buildbot
```
Comment (gibbed) [inline _config.yml]: No need to remove this.

File: _config.yml [hunk 4]
```diff
@@ -71,14 +68,11 @@ navigation:
...
-#      url: http://build.xenia.jp/waterfall
-#      icon: settings_system_daydream
 
 artifacts:
-  - title: Windows 7+ x64
+  - title: Windows 10 x64
```
Comment (gibbed) [inline _config.yml]: This is factually incorrect. It's Windows 7 or up. The title of the branch (later) clarifies why, well, that does it inaccurate too. If you want to reduce the noise here, then move the information from the branch title to the artifact title.

File: _includes/page-footer.html [hunk 5]
```diff
@@ -2,15 +2,6 @@
   <div class="wrapper">
 
     <div class="footer-col-wrapper">
-      <!--
```
Comment (gibbed) [inline _includes/page-footer.html]: No need to remove this.

File: download.md [hunk 6]
```diff
@@ -23,6 +23,6 @@ permalink: /download/
   {% endfor %}
 {% endfor %}
 
-Linux builds coming eventually...
+Linux builds coming Soon™
```
Comment (gibbed) [inline download.md]: No.

File: index.html [hunk 7]
```diff
@@ -33,9 +33,6 @@ <h3>
           <p class="updated" style="display:inline">
             Latest commit to the <strong>master</strong> branch on <span class="update-date">???</span>.
           </p>
-          <!--<a href="http://build.xenia.jp/waterfall" style="position: absolute; right: 10px; display: inline;">
```
Comment (gibbed) [inline index.html]: No need to remove this.

File: index.html [hunk 8]
```diff
@@ -76,17 +73,6 @@ <h2>
 <script src="{{ site.baseurl }}/scripts/merged-github-underscore.js"></script>
 
 <script>
-  /*
```
Comment (gibbed) [inline index.html]: No need to remove this.

File: _config.yml (hunk 4 above)
Comment (Margen67) [reply _config.yml]: What do you think of this? https://margen67.github.io/xenia/download/

**Pull request merged**